src/lib/eina/eina_debug_timer.c

```diff
@@ -69,24 +68,24 @@ _timer_append(Eina_Debug_Timer *t)
    unsigned int prev_time = 0;
    char c = '\0';
    EINA_LIST_FOREACH(_timers, itr, t2)
-     {
-        if (t2->timeout > t->timeout) goto end;
-        prev_time = t2->timeout;
-     }
+    {
+      if (t2->timeout > t->timeout) goto end;
+      prev_time = t2->timeout;
+    }
```

Comment:
Again, this is EFL default. You should keep the 2 -> { -> 3 spaces indent style.